Amazon Scraper (#885)
avinashkranjan/Amazing-Python-Scripts: Amazon Scraper/README.md

> ### Amazon Scraper

Reviewer: ```suggestion
# Amazon Scraper
```

Reviewer: This change is not done

# Amazon Item Scraper

> This is a python-amazon scraper is expected to work on any amazon.in link provided to it and will update the information accordingly.

Reviewer: ```suggestion
This is a python-amazon scraper which is expected to work on any amazon.in link provided to it and will update the information accordingly.
```

> In this scraper I have used the Requests library for the generating the requersts and the scientific libraries of BeautifuSoup and Pandas for generating the Data Frames.

Reviewer: ```suggestion
In this scraper I have used the Requests library for the generating the requests and the scientific libraries of BeautifulSoup and Pandas for generating the Data Frames.
```

> I have also used the time library so that the programm and auto update the stocks according to the user's wish.

Reviewer: ```suggestion
I have also used the time library so that the program and auto update the stocks according to the user's wish.
```
